apps/modernization-ui/src/pages/advancedSearch/components/PatientResults.tsx

```diff
@@ -3,6 +3,7 @@ import { useEffect, useRef, useState } from 'react';
 import { PersonName } from '../../../generated/graphql/schema';
 import { calculateAge } from '../../../utils/util';
 import '../AdvancedSearch.scss';
+import '../../../settings.scss';
```

Comment:
No need to import this scss file here as it's already imported into the index.tsx root file.